lib/InjectionException.php

```diff
@@ -15,6 +15,51 @@ public function __construct(array $inProgressMakes, $message = "", $code = 0, \E
     }
 
     /**
+     * Add a human readable version of the invalid callable to the standard 'invalid invokable' message.
+     * @param array $inProgressMakes
+     * @param $callableOrMethodStr
```

Comment:
It's actually not `string|callable`....It also includes `['classname', 'methodName']` and Auryn interprets this as "create an instance of that class and then call the method on it. That does not fit into `string|callable`.